src/admin/seeds/dashboard/dashboard.html

```diff
@@ -78,12 +78,12 @@
                 </div>
               </div>
             </div>
-            <div class="featureContainer" if.bind="selectedSeed.projectTokenBalance >= selectedSeed.seedRemainder && (selectedSeed.isClosed || selectedSeed.claimingIsOpen)">
+            <div class="featureContainer" if.bind="retrievableProjectTokenAmount != 0 && selectedSeed.projectTokenBalance >= selectedSeed.seedRemainder && (selectedSeed.isClosed || selectedSeed.maximumReached || selectedSeed.isDead)">
```

Comment:
do we really need this `selectedSeed.projectTokenBalance >= selectedSeed.seedRemainder` if we already are using this `retrievableProjectTokenAmount `

Are you sure this shouldn't be `retrievableProjectTokenAmount.gt(0)`?

Reply:
yes, I realized now.